amgut/db/patches/0032.sql

```diff
@@ -0,0 +1,369 @@
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Turks and Caicos Islands', 225);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Tuvalu', 226);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Uganda', 227);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Ukraine', 228);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'United Arab Emirates', 229);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'United Kingdom', 230);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'United States', 231);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'United States Minor Outlying Islands', 232);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Uruguay', 233);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Uzbekistan', 234);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Vanuatu', 235);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Venezuela', 236);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Viet Nam', 237);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Virgin Islands, British', 238);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Virgin Islands, U.S.', 239);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Wallis and Futuna', 240);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Western Sahara', 241);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Yemen', 242);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Zambia', 243);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Zimbabwe', 244);
+
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES
+(149, 'Unspecified', 0), (149, 'Yes', 1), (149, 'No', 2);
+
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES
+(152, 'Unspecified', 0),
+(152, 'I tend to be constipated (have difficulty passing stool) - Type 1 and 2', 1),
+(152, 'I tend to have diarrhea (watery stool) - Type 5, 6 and 7', 2),
+(152, 'I tend to have normal formed stool - Type 3 and 4', 3);
```

Comment:
Question - would it be more desirable to separate out the types, or leave it as is?